mm2src/mm2_main/src/rpc/lp_commands/lp_commands.rs

```diff
@@ -44,6 +45,7 @@ pub struct GetPublicKeyHashResponse {
     public_key_hash: H160Json,
 }
 
+// TODO: Return public_key_hash for all available and active unique pubkeys
 pub async fn get_public_key_hash(ctx: MmArc, _req: Json) -> GetPublicKeyRpcResult<GetPublicKeyHashResponse> {
```

Comment:
Same here, add this to a checklist to do in another PR. The enhancement can be to get a specific enabled coin `pubkeyhash` which is the enabled `db_id`.